.github/dependabot.yml

```diff
@@ -6,14 +6,6 @@ updates:
     schedule:
       interval: weekly
 
-  - package-ecosystem: npm
```

Comment:
This means we can't set both `dependencies` and `devDependencies`?

Reply:
We can allow both by removing..

```
    allow:
      - dependency-type: development
```

In that case, I'd just sticking with `auto` for the [versioning strategy](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#versioning-strategy--)

Reply:
Ah, I see. So, setting only `dependency-type: development` is okay.